docs/api/collection.md

```diff
@@ -2,10 +2,9 @@
 title: "Collection"
 summary: |
   `/api/collection` endpoints. By default, these endpoints operate on Collections in the 'default' namespace, which is
-    the one that has things like Dashboards and Cards. Other namespaces of Collections exist as well, such as the
-    `:snippet` namespace, (called 'Snippet folders' in the UI). These namespaces are completely independent hierarchies.
-    To use these endpoints for other Collections namespaces, you can pass the `?namespace=` parameter (e.g.
-    `?namespace=snippet`).
+  the namespace that has things like Dashboards and Cards. Other namespaces of Collections exist as well, such as the
```

Comment:
Is this change supposed to be in here? 👀

Reply:
I've updated it upstream in the API code.